src/components/pages/GlobalSettings/General/ApiEndpoint.tsx

```diff
@@ -36,13 +36,16 @@ const ApiEndpoint = () => {
 
             localStorage.setItem(config.apiEndpoint, targetUrl);
 
-            dispatch(setApiEndpoint(targetUrl));
-
             if (res.namespace) {
               dispatch(setNamespace(res.namespace));
             }
 
-            notificationCall('passed', 'API endpoint set up  successfully');
+            dispatch(setApiEndpoint(targetUrl));
+
+            setTimeout(() => {
+              notificationCall('passed', 'API endpoint set up  successfully');
+              form.resetFields();
+            });
```

Comment:
Is there a reason for these changes?

Reply:
Also a little improvement I was asked about. Should have create another PR for that